e2e/framework/util.go

```diff
@@ -0,0 +1,34 @@
+package framework
+
+import (
+	"strconv"
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetControlPlaneHostAndPort(cl client.Client) (string, int32, error) {
+	var infraCluster configv1.Infrastructure
+	namespacedName := client.ObjectKey{
+		Namespace: "openshift-cluster-api",
+		Name:      "cluster",
+	}
+
+	if err := cl.Get(ctx, namespacedName, &infraCluster); err != nil {
+		return "", 0, err
+	}
+
+	hostAndPort := strings.TrimPrefix(infraCluster.Status.APIServerURL, "https://")
```

Comment:
Is it always guaranteed that we get the APIServerURL with an `https` prefix? Or could it also be `http` or else?